src/components/aktivitetskrav/vurdering/VurderAktivitetskrav.tsx

```diff
@@ -36,46 +20,12 @@ export const VurderAktivitetskrav = ({
   aktivitetskrav,
   oppfolgingstilfelle,
 }: VurderAktivitetskravProps) => {
-  const [visVurderAktivitetskravModal, setVisVurderAktivitetskravModal] =
-    useState(false);
-  const [modalType, setModalType] = useState<ModalType>();
-  const visVurderingAktivitetskravModalForType = (modalType: ModalType) => {
-    setModalType(modalType);
-    setVisVurderAktivitetskravModal(true);
-  };
-
   return (
     <Panel className="mb-4 flex flex-col pt-4 pr-4 pb-8 pl-8">
-      <ButtonRow className="ml-auto">
-        {aktivitetskrav?.status !== AktivitetskravStatus.FORHANDSVARSEL && (
-          <Button
-            icon={<HourglassTopFilledIcon aria-hidden />}
-            variant="secondary"
-            size="small"
-            onClick={() => visVurderingAktivitetskravModalForType("AVVENT")}
-          >
-            {texts.avvent}
-          </Button>
-        )}
-        <Button
-          icon={<XMarkIcon aria-hidden />}
-          variant="secondary"
-          size="small"
-          onClick={() => visVurderingAktivitetskravModalForType("IKKE_AKTUELL")}
-        >
-          {texts.ikkeAktuell}
-        </Button>
-      </ButtonRow>
-      <FlexRow>
-        <Heading level="2" size="large">
-          {texts.header}
-        </Heading>
-        <FlexColumn className="ml-2" justifyContent={JustifyContentType.CENTER}>
-          <HelpText placement="right">
-            <BodyShort size="small">{texts.helptext}</BodyShort>
-          </HelpText>
-        </FlexColumn>
-      </FlexRow>
+      <StatusKnapperad aktivitetskrav={aktivitetskrav} />
+      <Heading level="2" size="large">
+        {texts.header}
+      </Heading>
```

Comment:
Lettere å lese noe slikt? 😊
```suggestion
      <Row>
        <AvventButton />
        <IkkeAktuellButton />
      </Row>
      <Heading level="2" size="large">
        {texts.header}
      </Heading>
```

Reply:
Dette er pent, men det er ikke nødvendigvis så rett frem dessverre 😕 Se https://github.com/navikt/syfomodiaperson/pull/1062#discussion_r1379326386